FWCore/ParameterSet/python/Config.py

```diff
@@ -1034,10 +1053,10 @@ def __repr__(self):
     def __getattr__(self,attr):
         return self[attr]
 
-class SubProcess(_ConfigureComponent,_Unlabelable):
```

Comment:
@Dr15Jones The reason this change was made is that, without it, Process.add_(SubProcess) did the wrong thing silently. With this change. Process.add_(SubProcess) fails noticeably.
I'm OK with reverting this change if we can either fix or disable Process.add_(SubProcess).
This needs discussion.